### PR TITLE
docs: Fix misunderstood and incorrect comments

### PR DESCRIPTION
**Pull request by jeongarmy**
- Add a period at the end of the sentence.
- Modify wrong indent or syntax.
- Use a verb for section title.
- Modify misunderstood comments.

### CODE REVIEW

**Comment by seinfra** [comment]
Target : [4d35bb5fc067de2646c51bf2dec6f3ee5114dd32] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [4d35bb5fc067de2646c51bf2dec6f3ee5114dd32] - Code Rule Check (C++) OK.

File: docs/HowToDevelopThingsWithTizenStudioForRT.md [hunk 1]
```diff
@@ -88,24 +88,24 @@ If you edit the SmartThings resources in your project later on, the Model Manage
 
 ## 4. Build the Project
 You must build your project before flashing or debugging it.  
-You can build a TizenRT project in two ways: 
--[using a batch build](#Using-Batch-Build)
--[build project command](#Using-Build-Project).
+You can build a TizenRT project in two ways:
```
Comment (jeongchanKim) [inline docs/HowToDevelopThingsWithTizenStudioForRT.md]: Please remove the last space.

File: docs/HowToUsePeripheral.md [hunk 2]
```diff
@@ -1,5 +1,5 @@
 # How to Use Peripheral
-TizenRT provides five parts to use peripheral. They are outlined in this document as shown:
+TizenRT provides five parts to use peripheral. This documents describes these five parts.
```
Comment (jeongchanKim) [inline docs/HowToUsePeripheral.md]: documents -> document

File: docs/HowToDevelopThingsWithTizenStudioForRT.md [hunk 3]
```diff
@@ -88,24 +88,24 @@ If you edit the SmartThings resources in your project later on, the Model Manage
 
 ## 4. Build the Project
 You must build your project before flashing or debugging it.  
-You can build a TizenRT project in two ways: 
--[using a batch build](#Using-Batch-Build)
--[build project command](#Using-Build-Project).
+You can build a TizenRT project in two ways:  
+-[use a batch build](#Use-Batch-Build)
```
Comment (zhouxinhe) [inline docs/HowToDevelopThingsWithTizenStudioForRT.md]:
```suggestion
- [use a batch build](#Use-Batch-Build)
```

File: docs/HowToDevelopThingsWithTizenStudioForRT.md [hunk 4]
```diff
@@ -88,24 +88,24 @@ If you edit the SmartThings resources in your project later on, the Model Manage
...
+-[build project command](#Use-Build-Project).
```
Comment (zhouxinhe) [inline docs/HowToDevelopThingsWithTizenStudioForRT.md]:
```suggestion
- [build project command](#Use-Build-Project)
```

**Comment by seinfra** [comment]
Target : [5e5c4a6570058a6ca7699f3044459f0b602a357d] - Code Rule Check OK.

**Comment by seinfra** [comment]
Target : [5e5c4a6570058a6ca7699f3044459f0b602a357d] - Code Rule Check (C++) OK.

**Pull request merged**